streams/idlharness.any.js

```diff
@@ -29,8 +29,9 @@ idl_test(
 
     try {
       const stream = new ReadableStream({
-        pull() {
-          self.readableStreamByobRequest = controller.byobRequest;
+        pull(c) {
+          self.readableStreamByobRequest = c.byobRequest;
+          c.enqueue(new Uint8Array(0));
```

Comment:
This might be a bit controversial... After enqueuing a chunk (or responding to the request), the original BYOB request becomes [invalidated](https://streams.spec.whatwg.org/#readable-byte-stream-controller-invalidate-byob-request). This means that the IDL test will only ever see `view` set to `null` (never an `ArrayBufferView`), and the `respond()` and `respondWithNewView()` methods will always throw an error.

Now, `null` *is* a valid value for `view`, and the IDL test never actually tries to *call* `respond()`, so an invalidated `ReadableStreamBYOBRequest` will *still* pass all of the tests. But perhaps it's not the *best* candidate for testing. 😛

Reply:
Do we really need to enqueue() here? We could just not await the reader.read() below and as far as I know everything would still work.

Reply:
Good point, we can just keep the pull pending indefinitely. We still have to wait for `pull()` to be called, but we can manually resolve a promise for that (like in [some other tests](https://github.com/web-platform-tests/wpt/blob/2e78e6d435f33d7c44e68978fe0b459a801c2ee8/streams/readable-byte-streams/general.any.js#L1984-L1987)).

Reply:
Thanks, I like this better.